frontend/src/components/sightings/sightings.tsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+import {
+  fetchSightings,
+  SightingResponse,
+} from "../../clients/internalApiClient";
+import { SightingCard } from "../sightingCard/sightingCard";
+
+export const Sightings: React.FunctionComponent = () => {
+  const [sightings, setSightings] = useState<SightingResponse[] | null>(null);
```

Comment:
Normally `undefined` is used for this kind of thing, and what's nice about state is that the empty brackets automatically imply "...or undefined" as the type!
```suggestion
  const [sightings, setSightings] = useState<SightingResponse[]>();
```